config.json

```
@@ -58,7 +58,7 @@
     { "country":  "india", "geoName": "India", "cities": ["mumbai", "bengaluru", "chennai", "ahmedabad", "pune", "kolkata", "hyderabad", "new-delhi", "lucknow", "jaipur"], "imageUrl": "https://upload.wikimedia.org/wikipedia/en/4/41/Flag_of_India.svg" },
     { "country":  "indonesia", "geoName": "Indonesia", "cities": ["jakarta", "bandung", "surabaya", "makassar", "medan", "semarang", "palembang", "depok"], "imageUrl": "https://upload.wikimedia.org/wikipedia/commons/9/9f/Flag_of_Indonesia.svg" },
     { "country":  "iran", "geoName": "Iran", "cities": ["tehran", "isfahan", "tabriz", "shiraz", "mashhad", "qom"], "imageUrl": "https://upload.wikimedia.org/wikipedia/commons/c/ca/Flag_of_Iran.svg" },
-    { "country":  "iraq", "geoName": "Iraq", "cities": ["baghdad", "mosul", "basra", "erbil", "najaf"], "imageUrl": "https://upload.wikimedia.org/wikipedia/commons/f/f6/Flag_of_Iraq.svg" },
+    { "country":  "iraq", "geoName": "Iraq", "cities": ["baghdad", "mosul", "basra", "erbil", "najaf", "sulaymaniyah", "duhok", "halabja"], "imageUrl": "https://upload.wikimedia.org/wikipedia/commons/f/f6/Flag_of_Iraq.svg" },
```

It would be a nice addtion if you could add Kirkuk.
```suggestion
    { "country":  "iraq", "geoName": "Iraq", "cities": ["baghdad", "mosul", "basra", "erbil", "najaf", "sulaymaniyah", "duhok", "halabja", "kirkuk"], "imageUrl": "https://upload.wikimedia.org/wikipedia/commons/f/f6/Flag_of_Iraq.svg" },
```